<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1777</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Main_stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>1500</width>
      <height>910</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QStackedWidget" name="Main_page1_stk">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="page1_piclabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>760</width>
          <height>560</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="page1_ua">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>0</y>
          <width>54</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="page1_la">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>54</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="page1_ra">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>290</y>
          <width>54</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="MainMenu_miBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>158</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>主界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainMenu_rtdBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>158</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>实时数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainMenu_dnBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>158</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结构诊断</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1777</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="actionlink_DB"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlink_DB">
   <property name="text">
    <string>link DB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
